### ISSUE TITLE
Module not found: Can't resolve '$paraglide/runtime.js'

### ISSUE THREAD

**Issue by amazfs** [opral/inlang-paraglide-js#266]
the nextjs wizard `npx @inlang/paraglide-next init` silently skips next.config.mjs if it already exists ( already used for another plugin) . This raises `Module not found: Can't resolve '$paraglide/runtime.js'`

this can be fixed editing manually next.config.mjs

**Comment by teyfix** [comment]
@amazfs Can you share what did you update in next.config.mjs? thx

**Comment by teyfix** [comment]
Okay, I was facing this issue while running Storybook with Next.js. 

### This is how I resolved this issue;

1. Update `tsconfig.json` to resolve "$paraglide/*"
```json
{
  "paths": {
    "$paraglide/*": ["./src/paraglide/*"]
  }
}
```

2. Add prebuild script for auto-generated paraglide files
```json
{
  "scripts": {
    "prestorybook": "bunx paraglide-js compile --project ./project.inlang",
    "storybook": "storybook dev -p 6006 --config-dir ./src/storybook",
  }
}
```

### Resulting PR:
![image](https://github.com/user-attachments/assets/6e287d20-434b-400c-a2e4-60463ed2bbfb)

**Comment by samuelstroschein** [comment]
Yep, the messages need to be compiled before running tests. Thanks @teyfix for the guide
